include/picongpu/simulation_defines/param/speciesAttributes.param

```diff
@@ -254,4 +254,15 @@ namespace picongpu
      */
     alias( exchangeMemCfg );
 
+    /** alias to specify the boundary condition for particles
+     *
+     * The default behavior if this alias is not given to a species is that the
+     * particles which leave the global simulation box were deleted.
+     *
+     * other options:
+     *   GuardHandlerCallPlugins - notify all plugins to handle leaving particles
+     *   and delete the particles after wards.
+     */
+    alias( boundaryCondition );
```

Comment:
Same here: I would suggest to rename this to something more pictoral like:
- `onParticleLeave`
- `onLeavingSimVolume`
- `whenLeavingSimVolume`